rules/docker-database.mk

```diff
@@ -29,5 +29,5 @@ $(DOCKER_DATABASE)_BASE_IMAGE_FILES += redis-cli:/usr/bin/redis-cli
 $(DOCKER_DATABASE)_FILES += $(SYSCTL_NET_CONFIG) $(SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT)
 $(DOCKER_DATABASE)_FILES += $(UPDATE_CHASSISDB_CONFIG_SCRIPT)
 
-SONIC_BUSTER_DOCKERS += $(DOCKER_DATABASE)
-SONIC_BUSTER_DBG_DOCKERS += $(DOCKER_DATABASE_DBG)
+SONIC_BULLSEYE_DOCKERS += $(DOCKER_DATABASE)
```

Comment:
These variables aren't actually defined; only the containers that are based on a Debian version that doesn't match the base image (i.e. Bullseye) need to be added into the `SONIC_*_DOCKERS` variable. Since the base image is Bullseye, there is no `SONIC_BULLSEYE_DOCKERS` or `SONIC_BULLSEYE_DBG_DOCKERS` variables.

Reply:
Sorry, I don't quite understand those definitions in the mk file. Did you suggest that I should delete this line since `SONIC_BULLSEYE_DOCKERS` or `SONIC_BULLSEYE_DBG_DOCKERS` doesn't exist?

Reply:
Yes. Neither of those variables exist.

Those version-specific variables (stretch, Jessie, buster) are defined in our build scripts for all older versions of Debian, since the containers may be based on an older version, and dependencies need to be correctly handled. Therefore, since bullseye is the current version, there's no `SONIC_BULLSEYE_DOCKERS` or `SONIC_BULLSEYE_DBG_DOCKERS`.